spdx-utils/src/main/kotlin/SpdxLicense.kt

```diff
@@ -479,6 +484,11 @@ enum class SpdxLicense(
          */
         const val NOASSERTION = "NOASSERTION"
 
+        /**
+         * The version of the license list.
+         */
+        const val LICENSE_LIST_VERSION = "3.9-4-g1a3102c"
+
```

Comment:
You cheated and did not regenerate the file :laughing:

Reply:
How do you arrive at this conclusion?

Reply:
Because `LICENSE_LIST_VERSION` was in the file twice, at the old and at the new location. But now it's fine.

Reply:
hehe, you notice too many things ;-)